sdk.build.gradle

```
@@ -40,11 +40,11 @@ android {
 }
 
 dependencies {
-    testImplementation 'androidx.test:runner:1.2.0'
-    testImplementation 'androidx.test.ext:junit:1.1.1'
-    testImplementation 'org.robolectric:robolectric:4.3.1'
+    testImplementation 'androidx.test:runner:1.5.2'
+    testImplementation 'androidx.test.ext:junit:1.1.5'
+    testImplementation 'org.robolectric:robolectric:4.10.3'
 
-    testImplementation 'androidx.test:rules:1.2.0'
+    testImplementation 'androidx.test:rules:1.5.0'
     testImplementation 'com.google.truth:truth:1.0'
```

truth:1.1.4 is now available. how about it updating?
see: https://truth.dev/

```suggestion
    testImplementation 'com.google.truth:truth:1.1.4'
```

---

@satsukies 
Please make a new PR. These updates are just for Android 14 SDK and JVM changes.

---

@jmatsu 
I see. My comment was off the maintain of PR.